enoslib/infra/enos_g5k/utils.py

```diff
@@ -23,7 +23,8 @@ def dhcp_interfaces(c_resources):
         nics = desc.get("_c_nics", [])
         nics_list = [nic for nic, _ in nics]
         ifconfig = ["ip link set %s up" % nic for nic in nics_list]
-        cmd = "%s ; dhclient %s" % (";".join(ifconfig), " ".join(nics_list))
+        dhcp = ["dhclient %s" % nic for nic in nics_list]
+	cmd = "%s ; %s" % (";".join(ifconfig), ";".join(dhcp))
```

Comment:
I suspect a wrong indentation here

Reply:
Yes!, sorry. The difference when using tab and spaces